### PR TITLE
Add text-to-video spaces, resources, schema and more

### PR DESCRIPTION
**Pull request by merveenoyan**
Screenshot 2023-07-04 at 17 30 54

### CODE REVIEW

File: tasks/src/text-to-video/about.md [hunk 1]
```diff
@@ -1,72 +1,38 @@
-Text-to-video models can generate videos from a given text prompt. They can be used in any application that requires generating temporally and spatially consistent sequence of images from text.  
-
-
 ## Use Cases
 
 ### Script-based Video Generation
-
 Text-to-video models can be used to create short-form video content from a provided text script. These models can be used to create engaging and informative marketing videos. For example, a company could use a text-to-video model to create a video that explains how their product works or that shows how their product can solve a customer's problem.
```
Comment (osanseviero) [inline tasks/src/text-to-video/about.md:4]:
```suggestion
Text-to-video models can be used to create short-form video content from a provided text script. These models can be used to create engaging and informative marketing videos. For example, a company could use a text-to-video model to create a video that explains how their product works.

```

File: tasks/src/text-to-video/about.md [hunk 2]
```diff
@@ -1,72 +1,38 @@
...
-
-
 ## Use Cases
 
 ### Script-based Video Generation
-
 Text-to-video models can be used to create short-form video content from a provided text script. These models can be used to create engaging and informative marketing videos. For example, a company could use a text-to-video model to create a video that explains how their product works or that shows how their product can solve a customer's problem.
 
-
 ### Content format conversion
```
Comment (osanseviero) [inline tasks/src/text-to-video/about.md:6]: Nice examples, but I'm not sure we have a model that can generate videos with quality for these. We might have some assistive models, but nothing ready to do safety protocols/disaster relief

File: tasks/src/text-to-video/about.md [hunk 3]
```diff
@@ -1,72 +1,38 @@
...
 
 
-
-
 ## Task Variants
-
 Text-to-video models have different variants based on inputs and outputs.
 
-
 ### Text-to-video Retrieval
```
Comment (osanseviero) [inline tasks/src/text-to-video/about.md:19]: Is this really using `text-to-video` models though? This is closer to semantic search/sentence similarity.

Comment (merveenoyan) [reply tasks/src/text-to-video/about.md:20]: I think you can use a text-to-video model in sort of feature extraction-y fashion and that's what they mention (sort of like semantic similarity)

File: tasks/src/text-to-video/about.md (hunk 2 above)
Comment (merveenoyan) [reply tasks/src/text-to-video/about.md:7]: I agree

File: tasks/src/text-to-video/about.md (hunk 3 above)
Comment (osanseviero) [reply tasks/src/text-to-video/about.md:17]: But this sounds more like a different task then (kinda like CLIP)

File: tasks/src/text-to-video/data.ts [hunk 4]
```diff
@@ -44,12 +51,24 @@ const taskData: TaskDataCustom = {
...
+		{
+			description: "An application that generates video from text.",
+			id:          "fffiloni/zeroscope",
+		},
+		{
+			description: "An application that generates video from image and text.",
+			id:          "TempoFunk/makeavid-sd-jax",
+		},
+	],
+	summary:       "Text-to-video is the task of generating videos from text. Text-to-video models can be used in any application that requires generating temporally and spatially consistent sequence of images from text. ",
```
[the remark this answers is not in the document]
Comment (merveenoyan) [reply tasks/src/text-to-video/data.ts:71]: Removing

**Pull request merged**